infra/nightly-resources/handlers.js

```diff
@@ -17,6 +17,22 @@ async function load() {
   initializeChart();
 }
 
+// Top-level load function for the llvm page
+async function load_llvm() {
+  GLOBAL_DATA.currentRun = await getBench("./");
```

Comment:
Is `currentRun` used anywhere else? Does it need to be in global data?
If you want to use it in other places in the future we could do this request on startup, perhaps that would make more sense?